src/pages/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useParams } from 'react-router-dom';
+import Button from '../../components/Button/Button';
+import {
+  SearchBarForm,
+  Heading,
+  SearchBarContainer,
+  Prefix,
+  Input,
+} from './SearchBar.style';
+
+function SearchBar(props) {
+  // Get the subreddit value from the URL
+  const [searchTerm, setSearchTerm] = useState(useParams().subreddit);
```

Comment:
That's interesting. Didn't know that's possible. Have you seen that somewhere? Otherwise I'd probably put it on a separate line

```
const { subreddit } = useParams();
const [searchTerm, setSearchTerm] = useState(subreddit);
```